build.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+set -e
+
+SHA=`git rev-parse --short --verify HEAD`
+
+echo "\newcommand{\YellowPaperVersionNumber}{$SHA}" > Version.tex
+
+pdflatex -interaction=errorstopmode -halt-on-error Paper.tex
+bibtex Paper
+pdflatex -interaction=errorstopmode -halt-on-error Paper.tex
+pdflatex -interaction=errorstopmode -halt-on-error Paper.tex
+pdflatex -interaction=errorstopmode -halt-on-error Paper.tex
```

Comment:
Moved the build part to #220 and fixed it there. That PR can be merged I guess right now.